### ISSUE TITLE
Folder issues

### ISSUE THREAD

**Issue by EliasSalom** [Fullstack-Alfanar/New-Homeworks-Neven#2]
why your react app is inside the folder it's must be inside the repository !!!

**Comment by nevensh** [comment]
hey Elias .
I put my react app in folder because this repository of all new homework's . 
new homework= new folder .
i can do all my new homework's in one react app ?

**Comment by EliasSalom** [comment]
no!!! it's should be the same app just you will add on this app new components, 
please fix this issue and update me and close the issue

**Comment by EliasSalom** [comment]
Hi @musiur it's nice to meet you, 
this is a learning area, and the student here still learning the first steps to becoming a full-stack engineer so it's true what you have been writing in the previous comment but in the first couple of steps the student needs to know the right way to use the repo then they can learn that,
